forge/db/models/Application.js

```diff
@@ -91,12 +92,129 @@ module.exports = {
                             })
                         }
 
+                        if (associationsLimit) {
+                            include.limit = associationsLimit
+                            include.order = [['mostRecentAuditLogCreatedAt', 'DESC'], ['updatedAt', 'DESC']]
+                            include.attributes = {
+                                include: [...include.attributes, [
+                                    literal(`(
+                                        SELECT "createdAt"
+                                        FROM "AuditLogs"
+                                        WHERE "AuditLogs"."entityId" = cast("Project"."id" as VARCHAR)
+                                        AND "AuditLogs"."entityType" = 'project'
+                                        ORDER BY "createdAt" DESC
+                                        LIMIT 1
+                                    )`),
+                                    'mostRecentAuditLogCreatedAt'
```

Comment:
I trialed left joining the AuditLog table, however I was unable to get Sequelize to run it in the same query, thus is was not possible to order the instances and devices by `mostRecentAuditLogCreatedAt`, see: https://github.com/FlowFuse/flowfuse/compare/feat-3398-home-screen...feat-3398-home-screen-left-join